bin/hab-origin

```diff
@@ -60,16 +48,15 @@ function download_sig_key() {
   # (@afiune) The last entry is the active key
   key_with_revision=$(find "$hab_cache_key_path" -name "$origin-20*.pub" -exec basename {} \; | cut -d'.' -f1 | tail -1)
 
-  if ! grep -qs SIG-SEC-1 "$hab_cache_key_path/${key_with_revision}.sig.key"
-  then
-    # Download the key from citadel
-    echo "$(citadel ${key_with_revision}.sig.key)" > "$hab_cache_key_path/${key_with_revision}.sig.key"
+  if grep -qs SIG-SEC-1 "$hab_cache_key_path/${key_with_revision}.sig.key"; then
+    echo "Downloading ${key_with_revision}.sig.key to ${hab_cache_key_path}"
+    citadel "${key_with_revision}.sig.key" > "$hab_cache_key_path/${key_with_revision}.sig.key"
   fi
 }
 
-case "$1" in
+case "${1:-}" in
   "download-sig-key")
-    download_sig_key $2
+    download_sig_key "$2"
```

Comment:
I think you'll need to make this `"${2:-}"` and deal with the null `$1` in `download_sig_key` or make this something like
```bash
${2:?origin argument required for download-sig-key.}
```